Clicker/Models/Socket.swift

```diff
@@ -39,6 +39,10 @@ class Socket {
         socket.on(clientEvent: .disconnect) { _, _ in
             self.delegate?.sessionDisconnected()
         }
+
+        socket.on(clientEvent: .error) { data, _ in
+            self.delegate?.sessionError()
```

Comment:
i think `sessionErrored` might be a better name to indicate that an error occurred